api/candlepin-api-spec.yaml

```diff
@@ -2691,347 +4315,435 @@ paths:
+          $ref: '#/components/responses/default'
+
+  /owners/{owner_key}/subscriptions:
+    get:
+      description: Retrieves a list of subscriptions for an owner
+      tags:
+        - Owner Subscription
+      operationId: getOwnerSubscriptions
+      security: []
+      x-java-response:
+        type: Iterable
+        isContainer: true
+      parameters:
+        - name: owner_key
           in: path
-          description: Content ID
+          description: The key of the owner
           required: true
           schema:
             type: string
-        - name: enabled
-          in: path
-          description: Content enabled flag
-          required: true
-          schema:
-            type: boolean
       responses:
         200:
-          description: Content added to a product
+          description:  A list of subscriptions for an owner
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/ProductDTO'
+                type: array
+                items:
+                  $ref: '#/components/schemas/SubscriptionDTO'
         400:
-          description: Invalid inputs, unable to add content
+          description: Owner key is null or empty
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/ExceptionMessage'
               example:
-                displayMessage: Invalid inputs, unable to add content
+                displayMessage: Owner key is null or empty
                 requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
-        403:
-          description: Product is locked. Unable to add content
+        404:
+          description: Owner key could not be found using the provided key
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+              example:
+                displayMessage: Owner key could not be found using the provided key
+                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
+        default:
+          $ref: '#/components/responses/default'
+
+    put:
+      description: |
+        Tickle an owner to have all of their entitlement pools synced with their
+        subscriptions. This method (and the one below may not be entirely RESTful,
+        as the updated data is not supplied as an argument.
+        This API call is only relevant in a top level hosted deployment where
+        subscriptions and products are sourced from adapters. Calling this in
+        an on-site deployment is just a no-op.
+      tags:
+        - Owner Subscription
+      operationId: refreshPools
+      security: []
+      parameters:
+        - name: owner_key
+          in: path
+          description: The key of the owner
+          required: true
+          schema:
+            type: string
+        - name: auto_create_owner
+          in: query
+          description: Auto create owner?
```

Comment:
Please remove ```?``` special character from description here & add meaningful description.

Reply:
Updated the description.